pennylane/ops/qubit.py

```diff
@@ -1693,6 +1783,9 @@ def _eigvals(cls, *params):
     def decomposition(D, wires):
         return [QubitUnitary(np.diag(D), wires=wires)]
 
+    def adjoint(self, do_queue=False):
+        raise NotImplementedError
```

Comment:
Actually this one should be easy to invert right?

```suggestion
    def adjoint(self, do_queue=False):
        return QubitUnitary(np.diag(self.data[0] ** (-1)), wires=wires)
```